assignment/tool/src/webapp/js/studentViewSubmission.js

```diff
@@ -24,10 +25,191 @@ ASN_SVS.confirmDiscardOrSubmit = function(editorInstanceName, attachmentsModifie
 	}
 };
 
+ASN.switchTimesheetTab = function( source )
+{
+ if( source == 'tabAssignment' )
+ {
+ 	document.getElementById('tabAssignmentCurrent').style.display = 'block';
+ 	document.getElementById('tabAssignmentRef').style.display = 'none';
+ 	document.getElementById('tabTimeSheetRef').style.display = 'block';
+ 	document.getElementById('tabTimeSheetCurrent').style.display = 'none';
+ 	document.getElementById('StudentAssignmentCurrent').style.display = 'block';
+ 	document.getElementById('StudentTimesheetCurrent').style.display = 'none';
```

Comment:
Not opposed to this method, just wondering if using a show/hide toggle CSS class and applying and removing that is an option.

Reply:
Thank you for your advice.  It is indeed a good choice.  It is already changed.